### PR TITLE
Add missing functions - nft

### PR DESCRIPTION
**Pull request by jakcinmarina**

### CODE REVIEW

**Comment by jakcinmarina** [comment]
I will add the rest of the NFT functions (prepareAddNftMinter, mintNftWithUri, mintMultipleNFTWithUri) after transactions will be done, I am missing some functions for this.

File: examples/kcs-example/src/app/kcs.nft.example.ts [hunk 1]
```diff
@@ -30,32 +39,112 @@ export async function kcsNftExample() {
...
+    authorAddresses: ['0x687422eEA2cB73B5d3e242bA5456b782919AFc85'],
+    provenance: true,
+    cashbackValues: ['0.5'],
+    fixedValues: ['0.5'],
+    fromPrivateKey: '0x05e150c73f1920ec14caa1e0b6aa09940899678051a78542840c2668ce5080c2',
+    nonce: 0,
+  })
+
+  const deployHash = await kcsSDK.nft.deployNFTSmartContract({
+    // TODO openapi bug
```
Comment (vlnevyhosteny) [inline examples/kcs-example/src/app/kcs.nft.example.ts:64]: Are there still some openapi bugs? I have regenerated code from new version of openapi.

Comment (jakcinmarina) [reply examples/kcs-example/src/app/kcs.nft.example.ts:64]: probably not, I have to update it as well

Comment (vlnevyhosteny) [reply examples/kcs-example/src/app/kcs.nft.example.ts:64]: It should be in master

**Review by vlnevyhosteny — APPROVED** [review]

**Comment by jakcinmarina** [comment]
Need to add `prepareAddNftMinter` and `getNftImage` functions

**Review by npwork — APPROVED** [review]

**Pull request merged**